File: copy-images.js
const fs = require('fs');
const path = require('path');

const publicDir = path.join(__dirname, 'public');

// Copy screenshot files with simpler names
try {
  fs.copyFileSync(
    path.join(publicDir, 'Screenshot 2025-09-07 at 10.34.30 PM.png'),
    path.join(publicDir, 'connectivity1.png')
  );
  
  fs.copyFileSync(
    path.join(publicDir, 'Screenshot 2025-09-07 at 10.36.32 PM.png'),
    path.join(publicDir, 'connectivity2.png')
  );
  
  fs.copyFileSync(
    path.join(publicDir, 'Screenshot 2025-09-07 at 10.36.55 PM.png'),
    path.join(publicDir, 'connectivity3.png')
  );
  
  console.log('Successfully copied all connectivity images!');
  console.log('Files created:');
  console.log('- connectivity1.png');
  console.log('- connectivity2.png');
  console.log('- connectivity3.png');
} catch (error) {
  console.error('Error copying files:', error.message);
}

File: copy_images.py
import shutil
import os

public_dir = '/Users/sagarsahu/Developer/dhawan-properties/public'
os.chdir(public_dir)

files_to_copy = [
    ('Screenshot 2025-09-07 at 10.34.30 PM.png', 'connectivity1.png'),
    ('Screenshot 2025-09-07 at 10.36.32 PM.png', 'connectivity2.png'),
    ('Screenshot 2025-09-07 at 10.36.55 PM.png', 'connectivity3.png')
]

for source, dest in files_to_copy:
    try:
        if os.path.exists(source):
            shutil.copy2(source, dest)
            print(f"✓ Copied {source} → {dest}")
        else:
            print(f"✗ Source file not found: {source}")
    except Exception as e:
        print(f"✗ Error copying {source}: {e}")

print("\nListing connectivity files:")
for f in os.listdir('.'):
    if f.startswith('connectivity'):
        print(f"- {f}")
